src/components/AppHeader/AppHeader.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ted_logo from './ted_logo.png';
+import './AppHeader.css';
+
+function AppHeader() {
+  return (
+    <header className="Header">
+      <img src={ted_logo} className="HeaderLogo-Main" alt="logo" />
+      <p className="HeaderLogo-Sub">OPEN<strong>SOURCE</strong></p>
```

Comment:
__*Preference*__
Mild accessibility gains here:
- image alt to be TED instead of logo. As presented a screen reader would read this as "Logo Open Source" which I don't think is your intent.
- wrapping the `img` and `p` here in an `h1` changing the byline to an `h2`